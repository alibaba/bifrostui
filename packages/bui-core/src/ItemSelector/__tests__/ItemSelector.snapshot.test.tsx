import { snapshotTest } from 'testing';

describe('ItemSelector snapshot', () => {
  snapshotTest('ItemSelector');
});
