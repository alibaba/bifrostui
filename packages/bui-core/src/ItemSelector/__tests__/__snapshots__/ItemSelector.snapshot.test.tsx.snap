// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemSelector snapshot ItemSelector demo snapshot 0 1`] = `
<div
  className="bui-item-selector"
  style={
    Object {
      "height": "500px",
    }
  }
>
  <div
    className="bui-scroll bui-scroll-view-scroll-y bui-item-selector-scroll-view-container tph"
    onScroll={[Function]}
    onTouchMove={[Function]}
    style={Object {}}
  >
    <div
      className="bui-item-selector-all-item"
    >
      <div
        className="bui-item-selector-list-container"
      >
        <div>
          <div
            className="select-item-title"
            id="CRT"
          >
            当前城市
          </div>
          <div
            className="select-item-buttons"
          >
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              上海
            </span>
          </div>
        </div>
        <div>
          <div
            className="select-item-title"
            id="POS"
          >
            定位城市
          </div>
          <div
            className="select-item-buttons"
          >
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              北京
            </span>
          </div>
        </div>
        <div>
          <div
            className="select-item-title"
            id="HOT"
          >
            热门城市
          </div>
          <div
            className="select-item-buttons"
          >
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              北京
            </span>
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              上海
            </span>
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              广州
            </span>
          </div>
        </div>
        <div>
          <div
            className="select-item-title"
            id="A"
          >
            A
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              安康
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              鞍山
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id="B"
          >
            B
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              巴音郭楞
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              巴中
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id="C"
          >
            C
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              沧州
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              长春
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id="D"
          >
            D
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              大理
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              大连
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`ItemSelector snapshot ItemSelector demo snapshot 0 2`] = `
<div
  className="bui-item-selector"
  style={
    Object {
      "height": "500px",
    }
  }
>
  <div
    className="bui-item-selector-title"
  >
    自定义Index
    <div
      className="bui-item-selector-btn-close"
      onClick={[Function]}
    >
      ✕
    </div>
  </div>
  <div
    className="bui-scroll bui-scroll-view-scroll-y bui-item-selector-scroll-view-container tph container-has-title"
    onScroll={[Function]}
    onTouchMove={[Function]}
    style={Object {}}
  >
    <div
      className="bui-item-selector-all-item"
    >
      <div
        className="bui-item-selector-list-container"
      >
        <div>
          <div
            className="select-item-title"
            id="FRT"
          >
            Fruit
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Bannar
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Apple
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Grape
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Pear
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Orange
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Pineapple
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Kiwi
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Watermelon
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id="TFC"
          >
            交通工具
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              汽车
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              飞机
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              火车
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              卡车
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              轮船
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              公共汽车
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              大巴士
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              自行车
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              徒步
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              滑板车
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              三轮车
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id="SIY"
          >
            Stationery
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Bag
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Ruler
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Pen
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Pencil
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Ruler
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Rubber
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Pencilcase
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Desk
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              Lamp
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id="Main"
          >
            Customer
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              自定义数据
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试1
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试2
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试3
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试4
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试4
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试4
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试4
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试4
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试4
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              测试4
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div
    className="bui-item-selector-index-container left-in item-index-has-title"
  >
    <ul
      className="bui-item-selector-index-list"
      onTouchMove={[Function]}
    >
      <li
        className="bui-item-selector-index-item"
        data-code="FRT"
        onClick={[Function]}
        style={
          Object {
            "maxHeight": "calc((NaNvh - 6vmin) / 4)",
          }
        }
      >
        果
      </li>
      <li
        className="bui-item-selector-index-item"
        data-code="TFC"
        onClick={[Function]}
        style={
          Object {
            "maxHeight": "calc((NaNvh - 6vmin) / 4)",
          }
        }
      >
        通
      </li>
      <li
        className="bui-item-selector-index-item"
        data-code="SIY"
        onClick={[Function]}
        style={
          Object {
            "maxHeight": "calc((NaNvh - 6vmin) / 4)",
          }
        }
      >
        文
      </li>
      <li
        className="bui-item-selector-index-item"
        data-code="Main"
        onClick={[Function]}
        style={
          Object {
            "maxHeight": "calc((NaNvh - 6vmin) / 4)",
          }
        }
      >
        Main
      </li>
    </ul>
  </div>
</div>
`;

exports[`ItemSelector snapshot ItemSelector demo snapshot 0 3`] = `
<div
  className="bui-item-selector"
  style={
    Object {
      "height": "500px",
    }
  }
>
  <div
    className="bui-item-selector-title"
  >
    选择Item
    <div
      className="bui-item-selector-btn-close"
      onClick={[Function]}
    >
      ✕
    </div>
  </div>
  <div
    className="bui-scroll bui-scroll-view-scroll-y bui-item-selector-scroll-view-container tph container-has-title"
    onScroll={[Function]}
    onTouchMove={[Function]}
    style={Object {}}
  >
    <div
      className="bui-item-selector-all-item"
    >
      <div
        className="bui-item-selector-list-container"
      >
        <div>
          <div
            className="select-item-title"
            id=""
          >
            当前城市
          </div>
          <div
            className="select-item-buttons"
          >
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              上海
            </span>
          </div>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            定位城市
          </div>
          <div
            className="select-item-buttons"
          >
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              北京
            </span>
          </div>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            热门城市
          </div>
          <div
            className="select-item-buttons"
          >
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              北京
            </span>
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              上海
            </span>
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              广州
            </span>
          </div>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            A
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              安康
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              鞍山
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            B
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              巴音郭楞
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              巴中
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            C
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              沧州
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              长春
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            D
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              大理
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              大连
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;

exports[`ItemSelector snapshot ItemSelector demo snapshot 0 4`] = `
<div
  className="bui-item-selector"
  style={
    Object {
      "height": "500px",
    }
  }
>
  <div
    className="bui-scroll bui-scroll-view-scroll-y bui-item-selector-scroll-view-container tph"
    onScroll={[Function]}
    onTouchMove={[Function]}
    style={Object {}}
  >
    <div
      className="bui-item-selector-all-item"
    >
      <div
        className="bui-item-selector-list-container"
      >
        <div>
          <div
            className="select-item-title"
            id=""
          >
            当前城市
          </div>
          <div
            className="select-item-buttons"
          >
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              上海
            </span>
          </div>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            定位城市
          </div>
          <div
            className="select-item-buttons"
          >
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              北京
            </span>
          </div>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            热门城市
          </div>
          <div
            className="select-item-buttons"
          >
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              北京
            </span>
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              上海
            </span>
            <span
              className="bui-item-selector-item"
              onClick={[Function]}
            >
              广州
            </span>
          </div>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            A
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              安康
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              鞍山
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            B
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              巴音郭楞
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              巴中
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            C
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              沧州
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              长春
            </li>
          </ul>
        </div>
        <div>
          <div
            className="select-item-title"
            id=""
          >
            D
          </div>
          <ul
            className="bui-item-selector-list"
          >
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              大理
            </li>
            <li
              className="bui-item-selector-list-item"
              onClick={[Function]}
            >
              大连
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</div>
`;
